daffodil-japi/src/test/resources/test/japi/mySchema5.dfdl.xsd

```diff
@@ -19,11 +19,12 @@
 <schema xmlns="http://www.w3.org/2001/XMLSchema"
   targetNamespace="http://example.com" xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
   xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:tns="http://example.com">
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:tns="http://example.com"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions" >
 
   <annotation>
     <appinfo source="http://www.ogf.org/dfdl/">
-      <dfdl:format ref="tns:GeneralFormat" />
+      <dfdl:format ref="tns:GeneralFormat" dfdlx:inlineWhiteSpace="allow"/>
```

Comment:
I would suggest no changing the behavior of existing tests. Create new tests specifically to check for these new entities.